crds/configmanagement_v1_configmanagement.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2023 Google LLC
```

Comment:
According to go/copyright we shouldn't change this / need to retain the earliest year that appeared in the file

Reply:
reverted